src/components/atoms/NcModal/NcModal.tsx

```diff
@@ -25,6 +26,7 @@ const NcModal: FC<NcModalProps> = ({
   modalTitle = "Modal title",
   isOpenProp,
   onCloseModal,
+  noCloseButton = false,
```

Comment:
noCloseButton ou onCloseButton?

Reply:
its noCloseButton, to remove the Button on top right. But i will change to hasCloseButton.